types/ldapjs/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for ldapjs 2.2
+// Type definitions for ldapjs 3.0
```

Comment:
It looks like there are other breaking changes in v3:
https://github.com/ldapjs/node-ldapjs/releases/tag/v3.0.0

In particular `SearchRequest.scope` (and probably SearchOptions.scope too?).
And `.object` accesors that may affect `SearchEntry.object` here.
And other deprecations here: https://github.com/ldapjs/messages/blob/master/lib/deprecations.js